### PR TITLE
Use Planck 2015 as the default cosmology, update references.

### PR DESCRIPTION
**Pull request by aconley**
* Closes [#5371]

### CODE REVIEW

**Comment by astrofrog** [comment]
I've made a new label 'whatsnew-needed' - to remind us of things that should be mentioned in the 'what's new' closer to the release.

File: CHANGES.rst [hunk 1]
```diff
@@ -22,6 +22,8 @@ New Features
 
 - ``astropy.cosmology``
 
+  - The default cosmology has been changed to Planck 2015
```
Comment (astrofrog) [inline CHANGES.rst]: Please add the PR number

File: astropy/cosmology/parameters.py [hunk 2]
```diff
@@ -24,10 +24,10 @@
 The list of cosmologies available are given by the tuple
 `available`. Current cosmologies available:
 
-Planck 15 parameters from Planck Collaboration 2015, arXiv: 1502.01589
+Planck 15 parameters from Planck Collaboration 2016, A&A, 594, A13
```
Comment (astrofrog) [inline astropy/cosmology/parameters.py]: Is it supposed to be ``Planck15`` at the start of the line? (to match the cosmology name)

Comment (astrofrog) [inline astropy/cosmology/parameters.py]: Same below for Planck13? (i.e. was it right before?)

File: CHANGES.rst (hunk 1 above)
Comment (aconley) [inline CHANGES.rst]: Always the problem -- you have to make a PR to get a PR number to put in your PR.

File: astropy/cosmology/parameters.py (hunk 2 above)
Comment (aconley) [inline astropy/cosmology/parameters.py]: they actually tend to get referred to in talks, etc. as Planck 2015, Planck 2013,
unlike, say, WMAP that used WMAP9, etc.  But we can't have a space in the
variable names.

I'll change them to be Planck 2015, Planck 2013

File: CHANGES.rst (hunk 1 above)
Comment (astrofrog) [inline CHANGES.rst]: Yes, we need a better solution for this :)

**Comment by aconley** [comment]
@astrofrog -- need anything else here?  I think I addressed both of the above.

**Comment by astrofrog** [comment]
@aconley - this looks good - but could you maybe add the 'what's new' entry now to save time later?

**Comment by aconley** [comment]
@astrofrog - Done.

**Comment by astrofrog** [comment]
@aconley - sorry, I think there was a misunderstanding - there are now two changelog entries - what I meant was that we need to add an entry in this page:

http://docs.astropy.org/en/latest/whatsnew/1.3.html

This is the page where we outline the main new features but also important backward-incompatible changes (hence why I think we should mention it there).

**Comment by aconley** [comment]
Aack, yes, sorry.

**Comment by aconley** [comment]
Should be fixed now, plus I squashed.

**Comment by astrofrog** [comment]
Thanks!

**Pull request merged**